stan/math/rev/scal/fun/round.hpp

```diff
@@ -16,7 +16,7 @@ namespace stan {
           op_v_vari(::round(avi->val_), avi) {
         }
         void chain() {
-          if (unlikely(boost::math::isnan(avi_->val_)))
+          if (unlikely(is_nan(avi_->val_)))
```

Comment:
I'm going to fix all these to have them call Boost's versions and throw exceptions, so this test will go away and get rolled into the constructor's call to `round()`.  But nothing to fix here now.